code/datums/diseases/black_goo.dm

```diff
@@ -146,10 +146,7 @@
 				target.AddDisease(new /datum/disease/black_goo)
 				to_chat(user, SPAN_XENOWARNING("<b>You sense your target is now infected.</b>"))
 
-	if(isSynth(target))
-		target.Slow(2)
-	else
-		target.Superslow(2) // Make them slower
+	target.apply_effect(2, SUPERSLOW)
```

Comment:
this was specifically made to only regular slow synths

Reply:
but... typechecks... waaaaa